modules/tinymce/src/themes/silver/main/ts/Theme.ts

```diff
@@ -44,20 +44,13 @@ export default (): void => {
     const renderUI = (): RenderResult => {
       const renderResult = renderModeUI();
 
-      const optScrollingContext = ScrollingContext.detectWhenSplitUiMode(
+      // If we have a scroll container, it looks like we need to calculate the bounds each time, just in
+      // case the scrolling context details have changed since the last time. The bounds considers
+      // the Boxes.box sizes, which might change over time.
+      popupSinkBounds = () => ScrollingContext.detectWhenSplitUiMode(
         editor,
         popups.getMothership().element
-      );
-      optScrollingContext.each(
-        (sc) => {
-          popupSinkBounds = () => {
-            // At this stage, it looks like we need to calculate the bounds each time, just in
-            // case the scrolling context details have changed since the last time. The bounds considers
-            // the Boxes.box sizes, which might change over time.
-            return ScrollingContext.getBoundsFrom(sc);
-          };
-        }
-      );
+      ).map(ScrollingContext.getBoundsFrom).getOr(Boxes.win());
```

Comment:
```suggestion
      ).fold(Boxes.win, ScrollingContext.getBoundsFrom);
```

Reply:
@TheSpyder  the code has been changed with the suggestion you wrote in this comment: https://github.com/tinymce/tinymce/pull/9725#discussion_r1665093904 so this part no long exist, let me know if the new implementation is ok